### ISSUE TITLE
Can't run react-server-test-pages under `npm2`

### ISSUE THREAD

**Issue by roblg** [redfin/react-server#662]
Repro:
* clone `react-server`
* run `npm run bootstrap` (succeeds)
* cd packages/react-server-test-pages
* `npm run start` (fails)

Results in many errors similar to:

```
ERROR in ../react-server-core-middleware/lib/coreCssMiddleware.js
Module not found: Error: Cannot resolve module 'babel-runtime/helpers/classCallCheck' in /Users/me/redfin/react-server/packages/react-server-core-middleware/lib
 @ ../react-server-core-middleware/lib/coreCssMiddleware.js 13:23-70
```

**Comment by gigabo** [comment]
Ugh.  Thanks for reporting @roblg.

**Comment by sresant** [comment]
This was fixed with #668
